### ISSUE TITLE
WCAG failure

### ISSUE THREAD

**Issue by tijascott** [iamstevendao/vue-tel-input#304]
I'm working on making a web app accessible use the WCAG guidelines. During an audit, an accessibility issue within this package was raised by the auditor. There is a WCAG 2.0 A failure on specification 4.1.1 Parsing.

https://www.w3.org/WAI/WCAG21/Understanding/parsing

The issue arises from a `<div>` being the child of a `<span>`

It has been identified as the flag inside the selection dropdown:

```
<div class ="vti__dropdown" tabindex="0">
  <span class="vti__selection">
    <div class="vti__flag us></div>
    <span class="vti__dropdown-arrow">▼</span>
  </span>
</div>
```

**Comment by tijascott** [comment]
From my understanding `span` is an inline-level element and `div` is a block-level element. It is out of spec if a block-level element is a child of an inline-level element. See: https://developer.mozilla.org/en-US/docs/Web/HTML/Inline_elements#conceptual_differences

It is suggested to either change the outside `span` (`<span class="vti__selection">`) to a block-level element or change the inside `div` (`<div class="vti__flag us></div>`) to an inline-level element.

Inline-level element list: https://developer.mozilla.org/en-US/docs/Web/HTML/Inline_elements#list_of_inline_elements
Block-level element list: https://developer.mozilla.org/en-US/docs/Web/HTML/Block-level_elements#elements

**Comment by tijascott** [comment]
#307 Fix here for vue3. Same fix can be applied to master branch

**Comment by tijascott** [comment]
Removed #307. #308 available for master
